rust-symcrypt/README.md

```diff
@@ -4,27 +4,36 @@ This crate provides friendly and idiomatic Rust wrappers over [SymCrypt](https:/
 
-For ease of use, we have included a `symcrypttestmodule.dll` and `symcrypttestmodule.lib` in the `bin/amd64/` folder for users on Windows. This will only work on computers using an `AMD64 (x86_64)` architecture.
+## Quick Start Guide
 
-We have also included the required `libsymcrypt.so` files needed for Linux users. These `.so` files have been built for the [Mariner](https://github.com/microsoft/CBL-Mariner) distro, but have been tested and confirmed working on `Ubuntu 22.04.3` via WSL. Support for other distros aside from Mariner is not guaranteed. 
+### Windows:
+Download the `symcrypt.dll` and `symcrypt.lib` for you corresponding CPU architecture from the [SymCrypt Releases Page](https://github.com/microsoft/SymCrypt/releases/tag/v103.4.2) and place them somewhere accessible on your machine.
 
-If you are using a different architecture, you will have to continue with the install and build steps outlined in the `BUILD.md` file.
+Set the required `SYMCRYPT_LIB_PATH` environment variable. You can do this by using the following command:
 
-## Quick Start Guide
+`setx SYMCRYPT_LIB_PATH "<your-path-to-symcrypt-lib-folder>"`
 
-### Windows:
-Copy the `symcrypttestmodule.dll` and `symcrypttestmodule.lib` from the `/bin/amd64` folder and place it into your `C:/Windows/System32` folder. 
+During runtime, Windows will handle finding all needed `dll`'s in order to run the intended program, this includes our `symcrypt.dll` file.
 
-For more information please see the `BUILD.md` file
+Here are 2 recommended options to ensure your `symcrypt.dll` is found by Windows during runtime, for more info please see [Dynamic-link library search order](https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order).
+
+1. Put the `symcrypt.dll` in the same folder as your output `.exe` file. If you are doing development (not release), the common path will be: `C:\your-project\target\debug\`.
+2. Permanently add the `symcrypt.dll` path into your System PATH environment variable. Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypt.lib`
+
+**Note this is a temporary solution while we work on shipping SymCrypt out of the box on Windows.**
+
+For more information please see the `BUILD.md` file on the [`rust-symcrypt`](https://github.com/microsoft/rust-symcrypt/tree/main/rust-symcrypt) page
 
 ### Linux:
-Copy all of the `libsymcrypt.so*` files from the `/bin/linux` folder and place it into your `/usr/bin/x86_64-linux-gnu/` folder. 
+Download and all of the `libsymcrypt.so*` files for you corresponding CPU architecture from the [SymCrypt Releases Page](https://github.com/microsoft/SymCrypt/releases/tag/v103.4.2).
+
+Support for `Debian` and `Ubuntu` via package manager is in the works, for now you must place the `libsymcrypt.so*` files into linker load path. The way that this is set will vary between distros. On most distros it set via the environment variable `$LD_LIBRARY_PATH`. 

```

Comment:
>Support for `Debian` and `Ubuntu` via package manager is in the works [](http://example.com/codeflow?start=0&length=69)

Maybe mention here as well that it is already available on Azure Linux via package manager

Reply:
Will note that.